pylops/utils/dottest.py

```diff
@@ -93,13 +93,13 @@ def dottest(
     # make u and v vectors
     rdtype = np.ones(1, Op.dtype).real.dtype
 
-    u = ncp.random.randn(nc).astype(rdtype)
+    u = randn(nc, backend=backend).astype(rdtype)
```

Comment:
The backend argument is now keyword-only in `randn`

Reply:
Sounds good :)